src/System.Web.HttpUtility/tests/HttpUtility/HttpUtilityTest.cs

```diff
@@ -412,6 +412,7 @@ public void UrlDecode(string decoded, string encoded)
 
         [Theory]
         [MemberData(nameof(UrlDecodeData))]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "Full framework assumes the current codepage is UTF8")]
```

Comment:
I do not. I very briefly looked into this test and it looked like a problem with the test assuming the codepage, but looking at it again I'm not certain that is the issue.